src/CloudLfs.Core.IntegrationTests/GitLfsBrokerTests.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Microsoft.MixedReality.CloudLfs.Brokers;
+using Microsoft.MixedReality.CloudLfs.Models;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.IO;
+using System.Security.Cryptography;
+using System.Threading.Tasks;
+
+namespace CloudLfs.Core.UnitTests
+{
+    [TestClass]
+    public class GitLfsBrokerTests
+    {
+        [TestMethod]
+        public async Task GitLfsBroker_CanGetLfsObject222()
+        {
+            // authenticate
+            var authBroker = new GitBroker("D:\\Repos\\mixedreality.platform.v3");
```

Comment:
we will probably need to make this a config or a temp folder, and if it doesn't exist then create it

Reply:
actually, should this be removed? or are we trying to decide whether we specify the folder or use CurrentDirectory?

Reply:
Will address in next PR. This should be a relative path.